README.md

```diff
@@ -1 +1,85 @@
-# fast-formdata-stringify
\ No newline at end of file
+# fast-formdata-stringify
+
+This module aims to serialize your data into `application/x-www-form-urlencoded` encode.
```

Comment:
encode -> enconding.

It's still not clear why I would like to do that. Which alternative options do I have?